### PR TITLE
add homepage

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Clockwork now has a homepage 

- home page contains a button to go to the task page

🔗  Preview: https://deploy-preview-4--clockwork-app.netlify.app/

### CODE REVIEW

File: src/components/home/_home.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    .hero-text {
+      font-size: 1.4rem;
+    }
+  }
+}
+
+@media screen and (min-width: 769px) {
+  .home-main {
+    margin-left: 257px;
```
Comment (prachi-sahani) [inline src/components/home/_home.scss:24]: you may consider keeping this property in relative units

**Review by prachi-sahani — COMMENTED** [review]
overall the code is clean and good to merge
suggestion - you can make the task button as the primary button

**Review by Tanay-J — APPROVED** [review]
Code is as clean as the UI. Good work.
Approving the changes.

File: src/components/home/_home.scss (hunk 1 above)
Comment (junaidshaikh-js) [reply src/components/home/_home.scss:24]: The absolute unit is used for the exactness with the width of the sidenavigation

**Comment by junaidshaikh-js** [comment]
> overall the code is clean and good to merge suggestion - you can make the task button as the primary button

Button has been changed to primary

**Pull request merged**